### PR TITLE
Release/0.1.2

### PR DESCRIPTION
**Pull request by oguzhanunlu**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -3,6 +3,18 @@ variable "name" {
   type        = string
 }
 
+variable "app_version" {
+  description = "Version of rdb loader redshift"
+  type        = string
+  default     = "5.8.0"
```
Comment (jbeemster) [inline variables.tf:9]: Same query as for transformer - does this version still work with the current config baked into this module?

**Review by jbeemster — APPROVED** [review]
LGTM - one quick Q and please add the CHANGELOG commit so we can get this merged!

File: variables.tf (hunk 1 above)
Comment (oguzhanunlu) [reply variables.tf:9]: thanks for the Q, yes it should work

Comment (oguzhanunlu) [reply variables.tf:9]: pushed the release commit!

**Review by jbeemster — APPROVED** [review]

**Pull request merged**